docs/docs/core-abilities/fetching_ticket_context.md

```diff
@@ -451,3 +452,49 @@ Name your branch with the ticket ID as a prefix (e.g., `ABC-123-feature-descript
     ```
     
     Replace `[ORG_ID]` with your Linear organization identifier.
+
+## Monday Integration 💎
+
+### Monday App Authentication
+The recommended way to authenticate with Monday is to connect the Monday app through the Qodo Merge portal.
+
+Installation steps:
+1. Go to the [Qodo Merge integrations page](https://app.qodo.ai/qodo-merge/integrations)
+2. Navigate to the **Integrations** tab
+3. Click on the **Monday** button to connect the Monday app
+4. Follow the authentication flow to authorize Qodo Merge to access your Monday workspace
+5. Once connected, Qodo Merge will be able to fetch Monday ticket context for your PRs
+
+### Monday Ticket Context
+`Ticket Context and Ticket Compliance is supported for Monday items, but not yet available in "PR to Ticket" feature.`
```

Comment:
/implement
Original: Ticket Context and Ticket Compliance is supported for Monday items, but not yet available in "PR to Ticket" feature.`
Fix: Corrected subject-verb agreement: Ticket Context and Ticket Compliance are supported for Monday items, but not yet available in the "PR to Ticket" feature.